usr/local/bin/bastille

```diff
@@ -126,11 +115,71 @@ Available Commands:
 
 Use "bastille -v|--version" for version information.
 Use "bastille command -h|--help" for more information about a command.
+Use "bastille -c|--config command" to specify a non-defaukt config file.
 
 EOF
     exit 1
 }
 
+bastille_conf_check
+bastille_perms_check
+
+BASTILLE_VERSION="0.13.20250126"
```

Comment:
It might be a matter of taste, but since this is static it should appear almost on the top of the file...